public/static/js/init.py

```diff
@@ -691,6 +715,8 @@ def debug_input(prompt=""):
 
     post_message({"cmd": "input"})
     resp = json.loads(synchronise('/@input@/req.js'))
+    if (js.workerInterrupted()):
+        raise KeyboardInterrupt()
```

Comment:
mis change. If user hits the stop button while waiting for an int(input()), we failed ungracefully, as we are stuck in an HTTP request and the keyboard interrupt doesn't always get picked up before the int(None) conversion. This helps to reduce the chance of that